src/main/webapp/WEB-INF/view/searchbar.jsp

```diff
@@ -24,6 +24,9 @@
                   a2 = document.createElement("a");
                   a2.style = "float: right";
                   a2.innerHTML = "add";
+                  a2.href = "#";
+                  a2.setAttribute("class", "add-user-button");
```

Comment:
What if the user is already added? This means we can add users more than once if we search, add, clear, then repeat.

Reply:
I was thinking of taking care of that on the server side(not adding it if the user is already there) but I could disable that button for users already in that conversation as well.

Reply:
The server does not have this context though, right? The client code is compiling the list of users to be added to the conversation, so only it knows about who's in that list, before it sends it off to the server.

Reply:
Wouldn't the server have access to the list of members in that conversation? I could disable the button for users already in that conversation just like it's disabled now if it's already selected then when it sends that list of names, I could add it to the lists of members.

Reply:
Oh, what I meant was the list of currently selected-for-addition users. Where you click the "Add" button and it queues it up? If you add a few users, clear the search, then search again, the "Add" buttons are all enabled for the new list, so you can add them again.

Reply:
Oh, sorry I misunderstood that. I'll find a way to fix that.

Reply:
It's not a simple problem given the current design of the search box, so feel free to discuss your ideas here. :)